druid/src/widget/flex.rs

```diff
@@ -674,7 +674,7 @@ impl<T: Data> Widget<T> for Flex<T> {
         for child in &mut self.children {
             match child {
                 Child::Fixed { widget, alignment } => {
-                    any_use_baseline &= *alignment == Some(CrossAxisAlignment::Baseline);
+                    any_use_baseline |= *alignment == Some(CrossAxisAlignment::Baseline);
```

Comment:
I think this makes sense (although I'm not super familiar with this code). But I'm confused by:
- why is `any_use_baseline` not updated in the "measure flex children" loop below?
- is there any way to even set alignment for a fixed child? I only see it in `FlexParams`.

Reply:
> * why is `any_use_baseline` not updated in the "measure flex children" loop below?

just updated the code for that. It now defaults to "false", and it is only set when a child uses baseline alignment.

> * is there any way to even set alignment for a fixed child? I only see it in `FlexParams`.

I guess not. But we can add this later if someone needs it?

Reply:
Sure, makes sense. I was just confused that we were only checking alignment for those children that *couldn't* have it set.